releases/sui/2025-01-ITS-v1.1.0.md

```diff
@@ -43,23 +43,22 @@ CHAIN=sui
 rm -rf ./sui/move
 ```
 
-- Deploy the ITS Contract and required depndencies on Sui (remember to save `sui/move` files for ITS deployment)
+- Deploy the ITS Contract and required dependencies on Sui (remember to save `sui/move` files for ITS deployment)
 
 ```bash
 node sui/deploy-contract.js deploy Abi
+# Deployer will be the owner of the contract
 node sui/deploy-contract.js deploy InterchainTokenService
 node sui/deploy-contract.js deploy Example
 ```
 
-### Register Sui chain on core
+### Register Sui ITS on ITS Hub
 
+#### Devnet:
 ```bash
 ITS_HUB_ADDR=$(cat ./axelar-chains-config/info/<your_config_name>.json | jq -rM '.axelar.contracts.InterchainTokenService.address')
 ITS_SUI_CHANNEL=$(cat ./axelar-chains-config/info/<your_config_name>.json | jq -rM '.chains.sui.contracts.InterchainTokenService.objects.ChannelId')
-```
 
-#### Devnet:
-```bash
 kubectl exec genesis-0 -- axelard tx wasm execute "$ITS_HUB_ADDR" \
```

Comment:
kubectl cmds shouldn't be included in public instructions. Only include axelard cmds or scripts. User can run it via kube if needed